### ISSUE TITLE
Stage doesn't load on specific machine

### ISSUE THREAD

**Issue by morantsur** [LLK/scratch-gui#215]
### Expected Behavior

When using Mac air, 2011, with OS X El Capitan, the stage should load. 

### Actual Behavior

It doesn't load. (It does work on Safari on the same machine, and chrome is updated)

Error msg: 
gui.js:68619 Uncaught TypeError: Cannot read property 'ARRAY_BUFFER' of null
    at createBufferFromTypedArray (https://llk.github.io/scratch-gui/gui.js:68619:23)
    at https://llk.github.io/scratch-gui/gui.js:68846:20
    at Array.forEach (native)
    at createAttribsFromArrays (https://llk.github.io/scratch-gui/gui.js:68840:26)
    at Object.createBufferInfoFromArrays (https://llk.github.io/scratch-gui/gui.js:69036:17)
    at RenderWebGL._createGeometry (https://llk.github.io/scratch-gui/gui.js:78166:37)
    at new RenderWebGL (https://llk.github.io/scratch-gui/gui.js:77209:15)
    at Stage.componentDidMount (https://llk.github.io/scratch-gui/gui.js:30516:29)
    at e.notifyAll (https://llk.github.io/scratch-gui/lib.min.js:25:4407)
    at r.close (https://llk.github.io/scratch-gui/lib.min.js:27:5627)
 
![image](https://cloud.githubusercontent.com/assets/15109618/23635380/71c707dc-029d-11e7-8e1c-1914d828f232.png)

### Steps to Reproduce
Try loading llk.github.io/scratch-gui
(It happens on LLK's "Jamaica" computer)

### Operating System and Browser
Mac air, 2011, with OS X El Capitan (10.11.4)
Chrome Version 56.0.2924.87 (64-bit)

**Comment by cwillisf** [comment]
I've seen a few cases where restarting Chrome would fix this, though we should definitely react better to this situation.

**Comment by paulkaplan** [comment]
@cwillisf should this be moved to scratch-render, since that is what is loading the webgl context and should probably be responsible for rescuing from errors creating/losing the context

**Comment by cwillisf** [comment]
@paulkaplan kinda. Last time @thisandagain and I talked about it, we came to the conclusion that a two-pronged approach would be best:
1. `scratch-render` should recognize when WebGL isn't available and react in a way that helps a developer, and
2. `scratch-gui` should recognize when WebGL isn't available and react in a way that helps an end-user.

We might implement step 2 in terms of step 1, but either way there's work to do in both repositories.
